server/routes/return-of-service.js

```diff
@@ -165,44 +107,4 @@ router.get(
   })
 );
 
-// Update return of service fields
-router.patch(
```

Comment:
What was the idea behind combining the endpoints? It's fairly typical to have separate endpoints for Create and Update. I think I prefer the duplication over the branching logic in the `POST` endpoint and it's not super correct to be modifying a record in a `POST` endpoint

Reply:
Yeah, I also prefer `PATCH`, but I wan't sure about it in this case. I thought we're creating a new entry and not editing it, isn't that right?
Do you think we can still keep `PATCH` endpoint in this case?

Reply:
I think we want the endpoints to match the conceptual action, the fact that we're using a transaction table shouldn't affect our endpoint structure

So data-wise we're adding a record but logically we're updating a user's return of service (this is my reasoning anyway, I do see the other side) @fw-noel What do you think?

Reply:
I need to look at the code from a higher level, but from first impressions, I think a POST seems fine, and is consistent with how things like participant statuses work